src/styles/blocks/_page.scss

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
```

Comment:
The universal selector (*) might cause performance issues, consider using a more specific selector or a CSS reset instead.